include/boost/url/params.hpp

```diff
@@ -346,12 +289,6 @@ class params
     std::size_t
     size() const noexcept;
 
-    /** <!-- @} --> */
-
-    /** <!-- @name Modifiers -->
-        <!-- @{ -->
-     */
-
     /** Clears the contents of the container
 
         Clears the contents of the container as if calling
```

Comment:
It looks bad when the description starts with the same word or words as the brief.